### PR TITLE
Improvement: 单据处理手动节点表单更换插件

### PR DESCRIPTION
**Pull request by zyxazhang**

### CODE REVIEW

File: frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue [hunk 1]
```diff
@@ -199,12 +199,19 @@
...
+                v-if="nodeInfo.type === 'APPROVAL'"
+                ref="fieldInfo"
+                :fields="nodeInfo.fields"
+                :all-field-list="allFieldList">
+              </field-info>
+              <process-form-plugin
+                v-else
+                ref="formField"
+                :fields="nodeInfo.fields">
+              </process-form-plugin>
```
Comment (luofann) [inline frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue:213]: 这里使用lesscode表单的逻辑是，如果当前单据是来自于lesscode应用提单，则人工节点表单用lesscode表单插件渲染

File: frontend/pc/.eslintignore [hunk 2]
```diff
@@ -9,5 +9,7 @@ src/assets/jsflow.js
 src/views/common/memberSelect
 src/views/slaManager/ticketStatus/firstStep.vue
 src/views/processManagement/processDesign/nodeConfigue/autoComponents/responseDataNode.vue
+src/assets/jsflow.js
```
Comment (luofann) [inline frontend/pc/.eslintignore:12]: 上面有个相同的

File: frontend/pc/src/main.js [hunk 3]
```diff
@@ -62,12 +62,16 @@ import locales from "element-ui/lib/locale";
 // view components
 import App from "./App";
 // components
-import router from "./router";
```
Comment (luofann) [inline frontend/pc/src/main.js:65]: 上面都是双引号，eslint没有报错吗

File: frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue [hunk 4]
```diff
@@ -665,6 +681,7 @@
           params,
           id,
         };
+        debugger;
```
Comment (luofann) [inline frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue:684]: 调试语句去掉

File: frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue [hunk 5]
```diff
@@ -591,6 +604,9 @@
           const params = {
             state_id: this.nodeInfo.state_id,
             fields: this.nodeInfo.fields.filter(ite => !ite.is_readonly && ite.showFeild).map(item => {
+              if (this.nodeInfo.type === 'NORMAL') {
+                item.value = this.$refs.formField.localValue[item.key];
+              }
```
Comment (luofann) [inline frontend/pc/src/views/ticket/details/currentSteps/CurrentStepItem.vue:609]: 如果使用lesscode表单，直接用v-model双向绑定可以取值

**Review by luofann — CHANGES_REQUESTED** [review]

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-itsm?pullRequest=902) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-itsm?pullRequest=902) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-itsm?pullRequest=902) it.</sub>

**Pull request closed without merging**